vllm_gaudi/ops/hpu_compressed_tensors.py

```diff
@@ -412,7 +404,7 @@ def transform_w_zp(x):
         else:
             self.w_zp_name: str = "qzeros"
             device = getattr(layer, self.w_q_name).device
-            groups = c.partition_weight_shape[0] // c.group_size
+            groups = c.partition_weight_shape[0] // c.group_size if c.group_size > 0 else 1
```

Comment:
can we make it:
It will be more clear?
```
groups = c.partition_weight_shape[0] // (c.group_size if c.group_size > 0 else 1)
```

Reply:
Hi Chendi, the group size should be 1 if per-channel.  I can change it to below for clarity

groups = (c.partition_weight_shape[0] // c.group_size) if c.group_size > 0 else 1